mlptrain/training/active.py

```diff
@@ -397,10 +397,29 @@ def _gen_active_config(config:      'mlptrain.Configuration',
     selector(traj.final_frame, mlp, method_name=method_name, n_cores=n_cores)
 
     if selector.select:
-        if traj.final_frame.energy.true is None:
-            traj.final_frame.single_point(method_name, n_cores=n_cores)
+        if selector.check:
+            logger.warning('distance selector, do backtracking')
```

Comment:
Yes it always performed with distance selector, changed to "currently applying distance selector, to avoid un-physical structures, do backtracking in the trajectory to find the first configuration in {selector.n_backtrack} steps recognised as outlier"